src/spikeinterface/widgets/motion.py

```diff
@@ -4,29 +4,31 @@
 
 from .base import BaseWidget, to_attr
 
+from spikeinterface.core import BaseRecording, SortingAnalyzer
+from spikeinterface.sortingcomponents.motion_utils import Motion
+
 
 class MotionWidget(BaseWidget):
     """
-    Plot the Motion object
+    Plot the Motion object.
 
     Parameters
     ----------
     motion : Motion
-        The motion object
-    segment_index : None | int
-        If Motion is multi segment, the must be not None
-    mode : "auto" | "line" | "map"
-        How to plot map or lines.
-        "auto" make it automatic if the number of depth is too high.
+        The motion object.
+    segment_index : int | None, default: None
+        If Motion is multi segment, the must be not None.
+    mode : "auto" | "line" | "map", default: "line"
+        How to plot map or lines. "auto" makes it automatic if the number of motion depths is too high.
```

Comment:
Just thinking about this (but fine to say out of scope for this PR, but "map" vs "line" isn't the clearest right? I mean "line" makes more sense, but thinking from a new user's perspective would I know what "map" meant. What do you think @JoeZiminski?

Reply:
I kind of like map, but we can be clearer about it's definition. Agree that it probably deserves a different issue

Reply:
I meant should we define "map" in all our docstrings, not change the argument :)